Periscope analysis

mplovepop opened the pull request:
Add Periscope Data analysis and readme to help with creating a DBT monitoring dashboard for Periscope.

mplovepop committed:
Add Periscope Data analysis and readme

In analysis/periscope/ folder

mplovepop committed:
Update README.md

Indent chart sublist

clrcrl commented:
Thanks @mplovepop! Can you do me a favor and add a screenshot of the dashboard this creates in the PR description? Over time we may add dashboards for Looker and Mode, and it would be great if we had a reference so that we can make them all consistent.

I intend to give this repo some love in the next few weeks, and will look at this as part of that!

emilieschario commented on analysis/periscope/models_out_of_sla.sql:
```diff
@@ -0,0 +1,17 @@
+    select
+      model
+      , max(deployment_started_at) as last_deployment_ts
+      , datediff('hour', max(deployment_started_at), getdate()) as hours_since_refreshed
```
I'm not sure what db you're on but isn't `CURRENT_DATE` more universal? I had to make that change here for Snowflake.

clrcrl replied:
This will be multi-db compatible
```suggestion
      , datediff('hour', max(deployment_started_at), {{ dbt_utils.current_timestamp() }}) as hours_since_refreshed
```

emilieschario reviewed (commented):
This was incredibly useful! +1 for getting this merged!

clrcrl commented:
@mplovepop just to remind myself, we had a discussion around SQL formatting, and this was the way that Periscope formats its SQL, right? As in, there's no point putting it into the [Fishtown style](https://github.com/fishtown-analytics/corp/blob/master/dbt_coding_conventions.md)?

clrcrl committed:
Update models_out_of_sla.sql timestamp

Add macro call dbt_utils.current_timestamp()
This will be multi-db compatible.

Co-Authored-By: mplovepop <michaelprentice@lovepopcards.com>

mplovepop commented:
> @mplovepop just to remind myself, we had a discussion around SQL formatting, and this was the way that Periscope formats its SQL, right? As in, there's no point putting it into the [Fishtown style](https://github.com/fishtown-analytics/corp/blob/master/dbt_coding_conventions.md)?

Correct. I don't think it matters much though, we can change to Fishtown style if you want. Periscope will just reformat when copy-pasting in and using their formatter command.
